core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -644,6 +681,27 @@ public void run() {
     }
 
     @Override
+    public void updateBalancingState(
+        final ConnectivityState newState, final SubchannelPicker newPicker) {
+      checkNotNull(newState, "newState");
+      checkNotNull(newPicker, "newPicker");
+      if (newState == SHUTDOWN) {
+        // It's unclear if it is appropriate to report SHUTDOWN state from lb. Ignore it for now.
```

Comment:
It's definitely inappropriate to report SHUTDOWN from LB. What's unclear is whether we should simply ignore it or do something else.